docs/docs/spec.md

```diff
@@ -258,11 +267,34 @@ Chunk files can be:
 
 Applications may choose to arrange chunks within files in different ways to optimize I/O patterns.
 
+### Transaction logs
+
+Transaction logs keep track of the operations done in a commit. They are not used to read objects
+from the repo, but they are useful for features such as commit diff and conflict resolution.
+
+Transaction logs are an optimization, to provide fast conflict resolution and commit diff. They are
+not absolutely required to implement the core Icechunk operations.
+
+Transaction log files are encoded using [flatbuffers](https://github.com/google/flatbuffers). The IDL for the
```

Comment:
This seems like an odd place to introduce and link to flatbuffers. Surely we have already mentioned it before, no? If not, move this earlier.

Reply:
Yes we only need to link out to flatbuffers once.